Dockerfile

```diff
@@ -88,8 +88,10 @@ RUN mkdir /.cache /.local
 ADD runtimes/python3-9 /workspace
 RUN chown -R 9653 /.cache /.local /workspace
 WORKDIR /workspace
+RUN apk add git
```

Comment:
I don't want to add git to this image. It increases the vuln surface and will create future work to keep patching.

Reply:
ok, so it looks like we really need this for pip